c7n/resources/s3.py

```diff
@@ -515,6 +520,41 @@ def __call__(self, b):
         return True
 
 
+@filters.register('no-access')
+class AccessDeniedFilter(Filter):
+    """Find buckets that Custodian can not access.
```

Comment:
this one can be removed, its just a value filter on the attribute. ie.

```
filters:
  - type: value
     key: "c7n:DeniedActions"
     op: in
     value_type: swap
     value: get_bucket_location
```

we can add a value type swap_set if you want the multi action matching per the doc string